### ISSUE TITLE
Batch Count stalls at 99%

### ISSUE THREAD

**Issue by cogfilms-3D** [vladmandic/automatic#1656]
### Issue Description

If I create a batch count of lets say 10 or more renders the system will sometimes get stuck on the last render at 99%.
No errors are reported in the  web interface or the CMD screen. The only way to get out of this is to close down vlad1111 and restart.

### Version Platform Description

```
Using VENV: S:\Vlad1111\automatic\venv
19:50:52-782267 INFO     Starting SD.Next
19:50:52-797775 INFO     Python 3.10.9 on Windows
19:50:52-898265 INFO     Version: a9fa014e Thu Jun 29 11:10:48 2023 +0900
19:50:53-411892 INFO     Latest published version: f0506fd51708b948759c667c7678418a76f624c9 2023-07-12T16:58:08Z
19:50:53-419877 INFO     nVidia CUDA toolkit detected
19:51:02-189846 INFO     Torch 2.0.1+cu118
19:51:02-288069 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
19:51:02-293557 INFO     Torch detected GPU: NVIDIA GeForce RTX 2060 SUPER VRAM 8192 Arch (7, 5) Cores 34
19:51:02-698686 INFO     Enabled extensions-builtin: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR',
                         'Lora', 'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-dynamic-thresholding',
                         'sd-extension-aesthetic-scorer', 'sd-extension-steps-animation', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'sd-webui-model-converter', 'seed_travel',
                         'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg', 'SwinIR']
19:51:02-734062 INFO     Enabled extensions: ['canvas-zoom', 'openpose-editor', 'sd-webui-openpose-editor']
19:51:02-752062 INFO     No changes detected: Quick launch active
19:51:02-781063 INFO     Extension preload: 0.0s S:\Vlad1111\automatic\extensions-builtin
19:51:02-785115 INFO     Extension preload: 0.0s S:\Vlad1111\automatic\extensions
19:51:02-831055 INFO     Server arguments: []
No module 'xformers'. Proceeding without it.
19:51:26-730928 INFO     Libraries loaded
19:51:26-736427 INFO     Using data path: S:\Vlad1111\automatic
19:51:26-748943 INFO     Available VAEs: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\VAE 2
19:51:27-320786 INFO     Available models: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\Stable-diffusion 39
19:51:32-106006 INFO     ControlNet v1.1.227
ControlNet v1.1.227
ControlNet preprocessor location: S:\Vlad1111\automatic\extensions-builtin\sd-webui-controlnet\annotator\downloads
19:51:32-812958 INFO     ControlNet v1.1.227
ControlNet v1.1.227
19:51:36-189915 INFO     Loading UI theme: name=black-orange style=Auto
Running on local URL:  http://127.0.0.1:7860
19:51:43-613331 INFO     Local URL: http://127.0.0.1:7860/
19:51:43-616832 INFO     Initializing middleware
Loading weights: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\Stable-diffusion\photon_v1.safetensors -- 0… -…
                                                                                                                  GB
19:51:45-504830 INFO     Setting Torch parameters: dtype=torch.float16 vae=torch.float16 unet=torch.float16
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading weights: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.safetensors
19:51:56-826903 INFO     Applying scaled dot product cross attention optimization
19:51:56-952989 INFO     Embeddings: loaded=0 skipped=0
19:51:56-964488 INFO     Model loaded in 12.5s (load=0.8s config=0.2s create=0.7s apply=7.3s vae=2.4s move=1.0s
                         embeddings=0.1s)
19:51:57-317919 INFO     Model load finished: {'ram': {'used': 3.23, 'total': 23.94}, 'gpu': {'used': 3.05, 'total':
                         8.0}, 'retries': 0, 'oom': 0} cached=0
19:51:57-438418 INFO     Startup time: 54.6s (torch=14.7s gradio=2.6s libraries=6.6s models=0.6s codeformer=0.2s
                         gfpgan=0.1s scripts=8.5s onchange=0.1s ui=7.1s launch=0.3s app-started=0.5s checkpoint=13.3s)
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:07<00:00,  2.68it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.61it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.67it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.68it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.65it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.65it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.64it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.64it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.62it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.63it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.62it/s]

```

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by vladmandic** [comment]
pls run with `--debug` to capture a bit more information

**Comment by cogfilms-3D** [comment]
OK I have run --debug and below is result. This crash dosn't happen allways it seems to be after a few hours of work. 
`Using VENV: S:\Vlad1111\automatic\venv
11:39:59-245396 INFO     Starting SD.Next
11:39:59-250897 INFO     Python 3.10.9 on Windows
11:39:59-316395 INFO     Version: a9fa014e Thu Jun 29 11:10:48 2023 +0900
11:39:59-727893 INFO     Latest published version: 6947776dcbcb81c75db83732479a5527cc7d614b 2023-07-12T20:59:02Z
11:39:59-731884 DEBUG    Setting environment tuning
11:39:59-734377 DEBUG    Torch overrides: cuda=False rocm=False ipex=False diml=False
11:39:59-738372 DEBUG    Torch allowed: cuda=True rocm=True ipex=True diml=True
11:39:59-744871 INFO     nVidia CUDA toolkit detected
11:40:01-685527 INFO     Torch 2.0.1+cu118
11:40:01-733037 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
11:40:01-737531 INFO     Torch detected GPU: NVIDIA GeForce RTX 2060 SUPER VRAM 8192 Arch (7, 5) Cores 34
11:40:01-953997 DEBUG    Repository update time: Thu Jun 29 03:10:48 2023
11:40:01-956498 DEBUG    Previous setup time: Tue Jul 11 23:41:31 2023
11:40:01-959015 INFO     Enabled extensions-builtin: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR',
                         'Lora', 'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-dynamic-thresholding',
                         'sd-extension-aesthetic-scorer', 'sd-extension-steps-animation', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'sd-webui-model-converter', 'seed_travel',
                         'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg', 'SwinIR']
11:40:01-973014 INFO     Enabled extensions: ['canvas-zoom', 'openpose-editor', 'sd-webui-openpose-editor']
11:40:01-978013 DEBUG    Latest extensions time: Tue Jul 11 23:41:07 2023
11:40:01-981517 DEBUG    Timestamps: version:1688004648 setup:1689115291 extension:1689115267
11:40:01-983520 INFO     No changes detected: Quick launch active
11:40:01-993999 INFO     Extension preload: 0.0s S:\Vlad1111\automatic\extensions-builtin
11:40:01-997998 INFO     Extension preload: 0.0s S:\Vlad1111\automatic\extensions
11:40:02-015513 DEBUG    Memory used: 0.33 total: 23.94 Collected 0
11:40:02-019004 DEBUG    Starting module: <module 'webui' from 'S:\\Vlad1111\\automatic\\webui.py'>
11:40:02-022011 INFO     Server arguments: ['--debug']
11:40:02-026509 DEBUG    Loading Torch
11:40:06-872290 DEBUG    Loading Gradio
11:40:08-114002 DEBUG    Loading Modules
No module 'xformers'. Proceeding without it.
11:40:08-715503 DEBUG    Loaded styles: styles.csv 0
11:40:10-558194 INFO     Libraries loaded
11:40:10-561203 DEBUG    Entering start sequence
11:40:10-726070 DEBUG    Version: {'app': 'sd.next', 'updated': '2023-06-29', 'hash': 'a9fa014e', 'url':
                         'https://github.com/vladmandic/automatic/tree/master'}
11:40:10-731072 INFO     Using data path: S:\Vlad1111\automatic
11:40:10-735587 DEBUG    Event loop: <_WindowsSelectorEventLoop running=False closed=False debug=False>
11:40:10-739576 DEBUG    Entering initialize
11:40:10-744572 INFO     Available VAEs: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\VAE 2
11:40:10-776588 INFO     Available models: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\Stable-diffusion 35
11:40:10-892575 DEBUG    Loading scripts
11:40:14-643326 INFO     ControlNet v1.1.227
ControlNet v1.1.227
ControlNet preprocessor location: S:\Vlad1111\automatic\extensions-builtin\sd-webui-controlnet\annotator\downloads
11:40:14-931833 INFO     ControlNet v1.1.227
ControlNet v1.1.227
11:40:16-646339 DEBUG    Scripts load: ['a1111-sd-webui-lycoris:1.816s', 'clip-interrogator-ext:0.219s', 'LDSR:0.082s',
                         'Lora:0.414s', 'sd-dynamic-thresholding:0.076s', 'sd-extension-aesthetic-scorer:0.14s',
                         'sd-extension-system-info:0.149s', 'sd-webui-agent-scheduler:0.535s',
                         'sd-webui-controlnet:0.586s', 'sd-webui-model-converter:0.08s',
                         'stable-diffusion-webui-images-browser:0.192s', 'stable-diffusion-webui-rembg:1.056s',
                         'SwinIR:0.086s', 'ScuNET:0.081s', 'openpose-editor:0.086s', 'sd-webui-openpose-editor:0.084s']
Scripts load: ['a1111-sd-webui-lycoris:1.816s', 'clip-interrogator-ext:0.219s', 'LDSR:0.082s', 'Lora:0.414s', 'sd-dynamic-thresholding:0.076s', 'sd-extension-aesthetic-scorer:0.14s', 'sd-extension-system-info:0.149s', 'sd-webui-agent-scheduler:0.535s', 'sd-webui-controlnet:0.586s', 'sd-webui-model-converter:0.08s', 'stable-diffusion-webui-images-browser:0.192s', 'stable-diffusion-webui-rembg:1.056s', 'SwinIR:0.086s', 'ScuNET:0.081s', 'openpose-editor:0.086s', 'sd-webui-openpose-editor:0.084s']
11:40:16-753323 INFO     Loading UI theme: name=black-orange style=Auto
11:40:16-761322 DEBUG    Creating UI
11:40:18-544339 DEBUG    Script: 1.06s ui_tabs
                         S:\Vlad1111\automatic\extensions-builtin\stable-diffusion-webui-images-browser\scripts\image_b
                         rowser.py
11:40:18-575327 DEBUG    Extensions list loaded: S:\Vlad1111\automatic\html\extensions.json
Running on local URL:  http://127.0.0.1:7860
11:40:22-051058 INFO     Local URL: http://127.0.0.1:7860/
11:40:22-055504 DEBUG    Gradio registered functions: 1832
11:40:22-057506 INFO     Initializing middleware
11:40:22-063003 DEBUG    Creating API
11:40:22-317503 DEBUG    Script: 0.16s app_started
                         S:\Vlad1111\automatic\extensions-builtin\sd-webui-agent-scheduler\scripts\task_scheduler.py
11:40:22-449019 DEBUG    Script: 0.12s app_started
                         S:\Vlad1111\automatic\extensions-builtin\sd-webui-agent-scheduler\scripts\task_scheduler.py
11:40:22-577367 DEBUG    Script: 0.12s app_started
                         S:\Vlad1111\automatic\extensions-builtin\sd-webui-agent-scheduler\scripts\task_scheduler.py
11:40:22-581367 DEBUG    Scripts setup: ['Tiled Diffusion:0.032s', 'ControlNet:0.052s', 'X/Y/Z grid:0.006s', 'Seed
                         travel:0.005s', 'Alternative:0.011s']
11:40:22-586367 DEBUG    Scripts components: []
11:40:22-589368 DEBUG    Model metadata: S:\Vlad1111\automatic\metadata.json no changes
11:40:22-935865 DEBUG    gc: collected=18111 device=cuda {'ram': {'used': 1.0, 'total': 23.94}, 'gpu': {'used': 1.0,
                         'total': 8.0}, 'retries': 0, 'oom': 0}
11:40:22-943866 DEBUG    Select checkpoint: photon_v1.safetensors [ec41bd2a82]
11:40:23-273366 DEBUG    gc: collected=124 device=cuda {'ram': {'used': 1.0, 'total': 23.94}, 'gpu': {'used': 1.0,
                         'total': 8.0}, 'retries': 0, 'oom': 0}
11:40:23-276370 DEBUG    Load model weights: existing=False
                         target=C:\stable-diffusion\MYSD\stable-diffusion-webui\models\Stable-diffusion\photon_v1.safet
                         ensors info=None
Loading weights: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\Stable-diffusion\photon_v1.safetensors -- 0… -…
                                                                                                                  GB
11:40:23-526367 DEBUG    Load model:
                         name=C:\stable-diffusion\MYSD\stable-diffusion-webui\models\Stable-diffusion\photon_v1.safeten
                         sors dict=True
11:40:23-530867 DEBUG    Verifying Torch settings
11:40:23-571365 DEBUG    Torch FP16 test passed
11:40:23-573867 DEBUG    Desired Torch parameters: dtype=FP16 no-half=False no-half-vae=False upscast=False
11:40:23-576866 INFO     Setting Torch parameters: dtype=torch.float16 vae=torch.float16 unet=torch.float16
11:40:23-579867 DEBUG    Torch default device: cuda
11:40:23-581867 DEBUG    Model dict loaded: {'ram': {'used': 1.03, 'total': 23.94}, 'gpu': {'used': 1.02, 'total':
                         8.0}, 'retries': 0, 'oom': 0}
11:40:23-602867 DEBUG    Model config loaded: {'ram': {'used': 1.03, 'total': 23.94}, 'gpu': {'used': 1.02, 'total':
                         8.0}, 'retries': 0, 'oom': 0}
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
11:40:24-142881 DEBUG    Model created from config: S:\Vlad1111\automatic\configs\v1-inference.yaml
11:40:24-145871 DEBUG    Model weights loading: {'ram': {'used': 1.98, 'total': 23.94}, 'gpu': {'used': 1.02, 'total':
                         8.0}, 'retries': 0, 'oom': 0}
Loading weights: C:\stable-diffusion\MYSD\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.safetensors
11:40:26-358867 DEBUG    Model weights loaded: {'ram': {'used': 5.18, 'total': 23.94}, 'gpu': {'used': 1.02, 'total':
                         8.0}, 'retries': 0, 'oom': 0}
11:40:27-394867 DEBUG    Model weights moved: {'ram': {'used': 3.19, 'total': 23.94}, 'gpu': {'used': 3.05, 'total':
                         8.0}, 'retries': 0, 'oom': 0}
11:40:27-409368 INFO     Applying scaled dot product cross attention optimization
11:40:27-443367 INFO     Embeddings: loaded=0 skipped=0
11:40:27-453366 INFO     Model loaded in 4.2s (load=0.2s config=0.1s create=0.5s apply=0.9s vae=1.3s move=1.0s)
11:40:27-793866 DEBUG    gc: collected=325 device=cuda {'ram': {'used': 3.22, 'total': 23.94}, 'gpu': {'used': 3.05,
                         'total': 8.0}, 'retries': 0, 'oom': 0}
11:40:27-802867 INFO     Model load finished: {'ram': {'used': 3.22, 'total': 23.94}, 'gpu': {'used': 3.05, 'total':
                         8.0}, 'retries': 0, 'oom': 0} cached=0
11:40:27-919368 INFO     Startup time: 25.9s (torch=4.8s gradio=1.2s libraries=2.4s vae=0.2s codeformer=0.1s
                         scripts=5.8s onchange=0.1s ui=5.0s launch=0.3s app-started=0.5s checkpoint=5.3s)
`

**Comment by cogfilms-3D** [comment]
I think this might be due to vram or memory usage as I had the system lockup this morning after trying outpaint operation on a small batch. The system froze and no message was displayed as to error. Here is the output
[sdnext.log](https://github.com/vladmandic/automatic/files/12049967/sdnext.log)
